### ISSUE TITLE
@mapbox/geojsonhint on npm has a syntax error in it

### ISSUE THREAD

**Issue by ckchow** [mapbox/geojsonhint#61]
In `package/geojsonhint.js` line 952 + context

```
errors.push({
                :qa

                    message: 'The geometries array in a GeometryCollection must contain only geometry objects',
                    line: geometryCollection.__line__
                });
```

The error is the ':qa' that appears in the middle of the line. Is this from the build process or some other error? The version of the package called just 'geojsonhint' doesn't have this problem.

**Comment by tmcw** [comment]
Bizarre, doesn't appear in the files from which that one is generated. Did I somehow insert this between prepublish & publish?

Well, anyway, I'll push 2.0.1 with the fix.

**Comment by tmcw** [comment]
Fixed in 2.0.1! Thanks for the report.
